lucille-core/src/test/java/com/kmwllc/lucille/indexer/SolrIndexerTest.java

```diff
@@ -275,7 +296,176 @@ public void testSolrIndexerWithMultipleCollections() throws Exception {
     assertEquals("doc4", collectionDocs.get("col2").get(1).getFieldValue(Document.ID_FIELD));
     assertFalse(collectionDocs.get("col2").get(0).containsKey(indexOverrideField));
     assertFalse(collectionDocs.get("col2").get(1).containsKey(indexOverrideField));
+  }
+
+  @Test
+  public void testSolrIndexerDelete() throws Exception {
+    String deletionMarkerField = "is_deleted";
+    String deletionMarkerFieldValue = "true";
+
+    Config config =
+        ConfigFactory.empty()
+            .withValue("indexer.batchSize", ConfigValueFactory.fromAnyRef(1))
+            .withValue(
+                "indexer.deletionMarkerField", ConfigValueFactory.fromAnyRef(deletionMarkerField))
+            .withValue(
+                "indexer.deletionMarkerFieldValue",
+                ConfigValueFactory.fromAnyRef(deletionMarkerFieldValue));
+    PersistingLocalMessageManager manager = new PersistingLocalMessageManager();
+
+    Document doc1 = Document.create("doc1", "test_run");
+    doc1.addToField(deletionMarkerField, deletionMarkerFieldValue);
```

Comment:
Should also create a doc that has a the deletionMarkerField populated with a value that's different from deletionMarkerFieldValue and confirm that this doc _isn't_ deleted.